File: shoring/flywheel/src/test/java/com/panopset/tests/gr/PipeTest.java
package com.panopset.tests.gr;

import java.io.IOException;

import org.junit.jupiter.api.Test;

import com.panopset.tests.transformer.GlobalReplaceTransformerTest;

public class PipeTest {

	private static final String REFRESH = "pipeTestRefresh.txt";
	private static final String TEMPLATE = "pipeTest.txt";
	private static final String EXPECTED = "pipeTestExpected.txt";

	@Test
	void test() throws IOException {
		new GlobalReplaceTransformerTest("red", "yellow", "blue", this.getClass().getPackageName(), REFRESH, TEMPLATE,
				EXPECTED).test();
	}
}

File: code/shoring/flywheel/src/test/kotlin/com/panopset/tests/gr/PipeTest.kt
package com.panopset.tests.gr

import com.panopset.tests.transformer.GlobalReplaceTransformerTest
import org.junit.jupiter.api.Test
import java.io.IOException

class PipeTest {
    @Test
    @Throws(IOException::class)
    fun test() {
        GlobalReplaceTransformerTest(
            "red", "yellow", "blue", this.javaClass.getPackageName(), REFRESH, TEMPLATE,
            EXPECTED
        ).test()
    }

    companion object {
        private const val REFRESH = "pipeTestRefresh.txt"
        private const val TEMPLATE = "pipeTest.txt"
        private const val EXPECTED = "pipeTestExpected.txt"
    }
}
